test/Performance/run-perf_tests.py

```diff
@@ -742,20 +748,14 @@ def parse_and_store_results_all( test_db, result_db, platform, start_time, mars,
     
     """
     print("Parsing and storing the results...")
-    parse_and_store_results( "sqlsrv-regular.xml", test_db, result_db, platform, "sqlsrv", start_time, mars, pooling )
-    parse_and_store_results( "sqlsrv-large.xml", test_db, result_db, platform, "sqlsrv", start_time, mars, pooling )
-    parse_and_store_results( "pdo_sqlsrv-regular.xml", test_db, result_db, platform, "pdo_sqlsrv", start_time, mars, pooling )
-    parse_and_store_results( "pdo_sqlsrv-large.xml", test_db, result_db, platform, "pdo_sqlsrv", start_time, mars, pooling )
+    parse_and_store_results( "sqlsrv-results.xml", test_db, result_db, platform, "sqlsrv", start_time, mars, pooling )
+    parse_and_store_results( "pdo_sqlsrv-results.xml", test_db, result_db, platform, "pdo_sqlsrv", start_time, mars, pooling )
  
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument( '-platform',         '--PLATFORM',         required=True, help='The name of the platform the tests run on' )
-    parser.add_argument( '-iterations',       '--ITERATIONS',       required=True, help='Number of iterations for regular tests', type=int )
-    parser.add_argument( '-iterations-large', '--ITERATIONS_LARGE', required=True, help='Number of iterations for regular tests',  type=int )
-    parser.add_argument( '-result-server',    '--RESULT_SERVER',    required=True, help='IP address of the Result Server' )
-    parser.add_argument( '-result-db',        '--RESULT_DB',        required=True, help='Name of the Result Database' )
-    parser.add_argument( '-result-uid',       '--RESULT_UID',       required=True, help='Username to connect to the Result Database' )
-    parser.add_argument( '-result-pwd',       '--RESULT_PWD',       required=True, help='Password to connect to the Result Database' )
+    parser.add_argument( '-platform',         '--PLATFORM',         required=True,  help='The name of the platform the tests run on' )
+    parser.add_argument( '-php-driver',       '--PHP_DRIVER',       default='both', help='Name of the PHP driver: sqlsrv, pdo_sqlsrv or both')
+    parser.add_argument( '-test-only',        '--TEST_ONLY',        default='all',  help='File name for only one test or all' )

```

Comment:
a bit nit-picking here but this name seems odd to me I would think it's just "-test"? Cos -test-only=all sounds strange. Just a suggestion, I don't feel strongly about it if you think it's better this way then leave it